virttest/virsh.py

```diff
@@ -3979,6 +3992,64 @@ def secret_set_value(uuid, password, options=None, encode=False, use_file=False,
     return command(cmd, **dargs)
 
 
+def nodedev_undefine(device_name, **dargs):
+    """
+    Return cmd result of removing a nodedev device definition
+
+    :param device_name: device name as listed by nodedev-list
+    :param dargs: standardized virsh function API keywords
+    :return: CmdResult object
+    """
+    cmd = "nodedev-undefine %s" % device_name
+
+    LOG.debug("Undefine the device %s", device_name)
```

Comment:
perhaps , we don't need to explicitly print out specific command information here since we have one consistent way : debug=True(by *dargs) to debug command and its output